lib/puppet/pops/validation/checker4_0.rb

```diff
@@ -361,6 +367,14 @@ def check_LiteralList(o)
     o.values.each {|v| rvalue(v) }
   end
 
+  def check_LiteralFloat(o)
+    migration_checker.report_ambiguous_float(o)
```

Comment:
Inconsistent use of `migration_checker` versus `@migration_checker` (on line 211)